src/main/java/org/eclipse/microprofile/starter/view/GeneratorDataBean.java

```diff
@@ -141,6 +146,16 @@ public void generateProject() {
         modelManager.prepareModel(model, false);
         creator.createArtifacts(model);
 
+        final LogData logData = new LogData.Builder()
+                .mpVersion(mpVersion)
+                .supportedServer(supportedServer)
+                .beansxmlMode(beansxmlMode)
+                .selectedSpecs(selectedSpecs)
+                .groupIdArtifacId(mavenData.getGroupId() + mavenData.getArtifactId())
+                .build();
+
+        new Thread(() -> DynamoDBLogger.log(logData)).start();
```

Comment:
@rdebusscher It works just because it is a static thing not depending on CDI.   If DynamoDBLogger is a CDI bean, it looses context in the new thread.  I don't know how to safely pass the CDI context to a new thread.
Obvious solution I am used from my EE days is EJB and async tasks / futures, but that would pull the whole EJB dependency here...